controllers/providers.ts

```diff
@@ -138,7 +138,7 @@ export const updateProvider = async (req: NextApiRequest, res: NextApiResponse)
           mutation: UPDATE_PROVIDER,
           variables: { ...fields, logoUrl, id },
         })
-        return res.status(204).json({
+        return res.status(200).json({
```

Comment:
Maybe 205 would be better, as you notice when the data changes we can reset UI

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/205